### DISCUSSION TITLE
[Windows] errno: -4048. Operation not permitted unlinking image from cache folder.

### DISCUSSION THREAD

**Discussion by amesas** [vercel/next.js#34508] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

Platform: win32
Arch: x64
Node: 14.18.1
Npm: 6.14.15
Next: 12.0.9
React: 17.0.2
React-Dom: 17.0.2
Sharp: 0.30.0

### What browser are you using? (if relevant)

N/A

### How are you deploying your application? (if relevant)

next start

### Describe the Bug

Next complains about not being able to delete images on cache folder.

[Error: EPERM: operation not permitted, unlink 'd:\example\.next\cache\images\hash\hash.webp']
error: -4048
code: 'EPERM'
syscall: 'unlink'
path: 'd:\example\.next\cache\images\hash\hash.webp'

Some of those images, fail to load with a error 500.

### Expected Behavior

Next should be able to delete it's own cache.

### To Reproduce

Windows OS.
Enable image optimization, install sharp.
next build
next start

**Comment by amesas** [comment]
Rare but only hapenning with webp and svg extensions, not with pngs.

**Comment by balazsorban44** [comment]
Hi, I see that you checked the ` I verified that the issue exists in Next.js canary release` checkbox, but you list 12.0.9 as the Next.js version. Have you tested `npm i next@canary`?

We would like to make sure the issue still exists or has already been fixed.

Could you put together a minimal reproduction so we could verify this behavior? :pray:

From a quick Google search, this might be a permission related issue on your Windows machine: https://stackoverflow.com/questions/34600932/npm-eperm-operation-not-permitted-on-windows

One suggested fix is running `npm cache clean`.

https://github.com/vercel/next.js/discussions/12380#discussioncomment-30170

Maybe also delete the `.next` folder as well.

Could you try that and report back?
